src/Microsoft.ML.KMeansClustering/KMeansPlusPlusTrainer.cs

```diff
@@ -94,35 +94,39 @@ public class Arguments : UnsupervisedLearnerInputBaseWithWeight
         /// <summary>
         /// Initializes a new instance of <see cref="KMeansPlusPlusTrainer"/>
         /// </summary>
-        /// <param name="env">The private instance of <see cref="IHostEnvironment"/>.</param>
+        /// <param name="env">The local instance of <see cref="IHostEnvironment"/>.</param>
```

Comment:
>The local instance of <see cref="IHostEnvironment"/> [](start = 30, length = 52)

Maybe update to: The environment to use.
So that it easier to understand.

Reply:
I think we are using this comment most everywhere. 

---
In reply to: [230873736](https://github.com/dotnet/machinelearning/pull/1524#discussion_r230873736) [](ancestors = 230873736)

Reply:
Actually, I don't think so. 'The local instance of IHostEnvironment' is a really bad comment for a public method.

---
In reply to: [231233831](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231233831) [](ancestors = 231233831,230873736)

Reply:
what would you suggest we use @Zruty0?

---
In reply to: [231367726](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231367726) [](ancestors = 231367726,231233831,230873736)

Reply:
What Artidoro said.

---
In reply to: [231386902](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231386902) [](ancestors = 231386902,231367726,231233831,230873736)

Reply:
that doesn't sound more more helpful to me than the current one.. but 2 to 1. 

---
In reply to: [231618928](https://github.com/dotnet/machinelearning/pull/1524#discussion_r231618928) [](ancestors = 231618928,231386902,231367726,231233831,230873736)